.ddev/commands/host/drowl-init-from-existing.sh

```diff
@@ -50,7 +50,7 @@ if [ $# -eq 0 ]; then
     y | Y | yes | Yes | YES)
       echo -e $'\e\n[32mOk! requiring development tools...\n\e[0m'
       ddev composer require --dev cweagans/composer-patches szeidler/composer-patches-cli drupal/admin_toolbar drupal/backup_migrate drupal/examples drupal/stage_file_proxy drupal/devel drupal/devel_debug_log drupal/devel_php drupal/coder drupal/examples
-      ddev composer require --dev drupal/core-dev:^10 drush/drush phpunit/phpunit:^9.5 phpspec/prophecy-phpunit phpstan/phpstan mglaman/phpstan-drupal phpstan/phpstan-deprecation-rules phpstan/phpstan-phpunit phpstan/extension-installer -W
```

Comment:
This seems unrelated? There probably was a reason to have an explicit version set here.